docs/pages/guides/best-practices/dividing-into-systems.mdx

```diff
@@ -0,0 +1,161 @@
+import { Callout } from "nextra/components";
+
+# Dividing Code into `System`s
+
+It is usually simplest to keep all the code that needs the same permissions and lives in the same namespace in a single `System`.
+This page discusses situations in which either you need to divide your code into multiple `System`s, or it seems like that would be a good solution.
+
+## Good reasons
+
+There are several reasons to divide the code in a namespace into multiple `System`s.
+
+### Access Control
+
+Each `System` is either publicly accessible (can be called by anybody) or accessible only from authorized addresses.
+These addresses are can be:
+
+- [Externally owned accounts](https://ethereum.org/en/developers/docs/accounts/#types-of-account)
+- Contracts outside of the `World`
+- `System`s that are _not_ in the root namespace
```

Comment:
this one is a little confusing because systems in the root namespace can also call other systems (including the private ones), it's just requires a different implementation

Reply:
Also, does access control happen in those calls, or are root `System`s allowed to call any function on any `System`, public or private?